MediaBrowser.WebDashboard/dashboard-ui/bower_components/emby-webcomponents/playback/experimentalwarnings.js

```diff
@@ -20,15 +20,15 @@ define(["connectionManager", "globalize", "userSettings", "apphost"], function(c
     }
 
     function showBlurayMessage() {
-        return showMessage("Playback of Bluray folders in this app is experimental. Some titles may not work at all. For a better experience, consider converting to mkv video files, or use an Emby app with native Bluray folder support.", "blurayexpirementalinfo", "nativeblurayplayback")
+        return showMessage("Playback of Bluray folders in this app is experimental. Some titles may not work at all. For a better experience, consider converting to mkv video files, or use an Jellyfin app with native Bluray folder support.", "blurayexpirementalinfo", "nativeblurayplayback")
```

Comment:
you have to replace "an Emby" with "a Jellyfin" to be correct on the article side :)